BruTile/Extensions/HttpTileSourceExtensions.cs

```diff
@@ -21,7 +20,7 @@ public static async Task<byte[]> GetTileAsync(this HttpTileSource tileSource, Ht
     {
         var bytes = tileSource.PersistentCache.Find(tileInfo.Index);
         if (bytes != null) return bytes;
-        bytes = await httpClient.GetByteArrayAsync(tileSource.GetUri(tileInfo)).ConfigureAwait(false);
+        bytes = await httpClient.GetByteArrayAsync(tileSource.GetUrl(tileInfo)).ConfigureAwait(false);
```

Comment:
Renamed IHttpTileSource to GetUrl instead of GetUri, but it is really a URL not a URN, because you can not request data from a URN.